Library/ViewModels/DiscoveryPageViewModel.swift

```diff
@@ -437,18 +437,6 @@ public final class DiscoveryPageViewModel: DiscoveryPageViewModelType, Discovery
       AppEnvironment.current.koala.trackEditorialHeaderTapped(refTag: refTag)
     }
 
-    personalizationCellTappedAndRefTag
-      .observeValues { refTag in
-        let attributes = optimizelyUserAttributes(refTag: refTag)
-
-        try? AppEnvironment.current.optimizelyClient?.track(
-          eventKey: "Editorial Card Clicked",
```

Comment:
Removed because the campaign is over right?

Reply:
This PR removes all events previously tracked on the Optimizely SDK, because those events have been migrated to the data lake. An upcoming PR will add the necessary Optimizely properties to the required events.